### PR TITLE
Validator: Configurable error message + more flexible country option

### PR DESCRIPTION
**Pull request by haffla**
Hello,

I added the possibility to pass a message to the `telephone_number` validator like all the standard Rails validators have. While I was at it I made the `country` option of the validator accept Strings and Symbols, too.

Thanks for this lib!

### CODE REVIEW

File: lib/active_model/telephone_number_validator.rb [hunk 1]
```diff
@@ -1,9 +1,23 @@
 class TelephoneNumberValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
-    country = options[:country].call(record) if options.key?(:country)
     valid_types = options.fetch(:types, [])
-    args = [value, country, valid_types]
+    args = [value, country(record), valid_types]
+    record.errors.add(attribute, message) if TelephoneNumber.invalid?(*args)
```
Comment (akfernun) [inline lib/active_model/telephone_number_validator.rb]: Can we add a test for the `message` functionality to ensure that when the record is invalid, the correct message is added to the errors object?

Comment (haffla) [reply lib/active_model/telephone_number_validator.rb]: Oh yes, I kind of overlooked that. Of course I will. Thank you!

Comment (haffla) [reply lib/active_model/telephone_number_validator.rb]: should be good now!

**Comment by akfernun** [comment]
Hi @haffla, 

I went ahead and merged your PR but have not yet cut a new release. Before releasing a new version, I'd like to get metadata for the validations updated to the latest version as well. I'll work on getting this done this week and will cut a new version of this gem at that time. Thanks again for your submission!

**Pull request merged**